apps/vor/pkg/lessonplan/store.go

```diff
@@ -0,0 +1,7 @@
+package lessonplan
+
+import "github.com/chrsep/vor/pkg/postgres"
+
+type Store interface {
+	CreateLessonPlan(input postgres.PlanData) (*postgres.LessonPlan, error)
```

Comment:
It's better to have no dependency on Postgres here.

Reply:
you mean for the input?